ansible/monitoring.yml

```diff
@@ -76,6 +76,18 @@
   hosts: prometheus
   tags: prometheus
   tasks:
+    - name: Check for existing prometheus binaries
+      stat:
+        path: /usr/local/bin/{{ item }}
+      register: prometheus_binaries
+      loop:
+        - prometheus
+        - promtool
+    - name: Skip prometheus install if prometheus binaries exist and prometheus_version not defined
+      # i.e. if prometheus_version isn't defined we don't care, so use what's already there
+      set_fact:
+        prometheus_skip_install: "{{ false if prometheus_version is defined else true }}" 
```

Comment:
Could add `when: prometheus_skip_install is not defined` to allow to it to be punched through directly from `group_vars` if its desirable for this to be able to be set directly?